Frontend.Integration.Tests/Pages/Projects/LegalRequirements/TrustAgreementTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Data.Models;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Data.TRAMS.ExtensionMethods;
+using Xunit;
+using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+
+namespace Frontend.Integration.Tests.Pages.Projects.LegalRequirements
+{
+    public class TrustAgreementIntegrationTests : BaseIntegrationTests
```

Comment:
Should there be a test for submitting the form?

Reply:
Yeah, sounds good. Added.